cypress/integration/html-reader/navigations.ts

```diff
@@ -9,18 +9,18 @@ describe('navigating an EPUB page', () => {
     cy.findByRole('link', { name: 'Return to Homepage' }).should(
       'have.prop',
       'href',
-      '/'
+      `${Cypress.config().baseUrl}/`
```

Comment:
This href is expecting the full path, so I added the base url since that's what it is expecting